src/components/activity-groups-list/activity-group-item/activity-item/baking-rewards.tsx

```diff
@@ -39,7 +39,7 @@ export const BakingRewards: FC<ActivityItemProps> = ({ activity }) => {
       timestamp={activity.timestamp}
     />
   );
-};
+});
```

Comment:
I'm afraid `isEqual` function as the second argument is necessary here because two `tokensDeltas` values may be 'deeply' equal but not strictly equal